### PR TITLE
Canto fixes

### PR DESCRIPTION
**Pull request by 0xtemujin**

### CODE REVIEW

**Comment by 0xtemujin** [comment]
image

File: src/static/js/canto_dex.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
...ion" }, { "inputs": [{ "internalType": "address", "name": "", "type": "address" }], "name": "isPair", "outputs": [{ "internalType": "bool", "name": "", "type": "bool" }], "stateMutability": "view", "type": "function" }, { "inputs": [], "name": "isPaused", "outputs": [{ "internalType": "bool", "name": "", "type": "bool" }], "stateMutability": "view", "type": "function" }, { "inputs": [], "name": "pairCodeHash", "outputs": [{ "internalType": "bytes32", "name": "", "type": "bytes32" }], "stateMutability": "pure", "type": "function" }, { "inputs": [], "name": "pauser", "outputs": [{ "internalType": "address", "name": "", "type": "address" }], "stateMutability": "view", "type": "function" }, { "inputs": [], "name": "pendingPauser", "outputs": [{ "internalType": "address", "name": "", "type": "address" }], "stateMutability": "view", "type": "function" }, { "inputs": [{ "internalType": "bool", "name": "_state", "type": "bool" }], "name": "setPause", "outputs": [], "stateMutability": "nonpayable", "type": "function" }, { "inputs": [{ "internalType": "address", "name": "_pauser", "type": "address" }], "name": "setPauser", "outputs": [], "stateMutability": "nonpayable", "type": "function" }]
```
Comment (zjesko) [inline src/static/js/canto_dex.js]: Lets be consistent with the variable names?

Eg. rename `CANTO_FACTORY_ABI` to `BaseV1Factory_ABI`
same with `CTOKEN_ABI_CANTO`

**Review by zjesko — CHANGES_REQUESTED** [review]
small fix only, rest lgtm

File: src/static/js/canto_dex.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+        prices[k.toLowerCase()] = prices[k];
+    }
+
+    const rewardTokenTicker = "WCANTO";
+    const poolCount = ctokenAddresses.length;
+    const BaseV1Router = new ethers.Contract("0xa252eee9bde830ca4793f054b506587027825a8e", BaseV1Router_ABI, App.provider);
+    const Comptroller = new ethers.Contract(ComptrollerAddress, Comptroller_ABI, App.provider);
+
+    for (i = 0; i < poolCount; i++) {
+        var Ctoken = new ethers.Contract(ctokenAddresses[i], CTOKEN_ABI, App.provider);
```
Comment (zjesko) [inline src/static/js/canto_dex.js:66]: This CTOKEN_ABI variable is not defined anywhere? are you sure this is working?

Also change the variable name

**Review by zjesko — CHANGES_REQUESTED** [review]

File: src/static/js/canto_dex.js (hunk 2 above)
Comment (0xtemujin) [reply src/static/js/canto_dex.js:66]: CTOKEN_ABI is already defined in the src/static/data.js of this project

**Review by zjesko — APPROVED** [review]

**Pull request merged**